sql-ballerina/tests/resources/sql/query/complex-test-data.sql

```diff
@@ -235,3 +235,20 @@ INSERT INTO ArrayTypes2 (row_id, int_array, long_array, float_array, double_arra
 INSERT INTO ArrayTypes2 (row_id, int_array, long_array, float_array, double_array, decimal_array, boolean_array, bit_array, string_array, blob_array, smallint_array, numeric_array, real_array, char_array, varchar_array, nvarchar_array, date_array, time_array, datetime_array, timestamp_array, time_tz_array, timestamp_tz_array)
   VALUES (4, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null, Null);
 
+CREATE TYPE INT_TYPE AS INT;
```

Comment:
Even though this is considered custom type in the SQL, in ballerina perspective this is just an integer type

Reply:
Yes as per the documentation here

http://hsqldb.org/doc/guide/databaseobjects-chapt.html#dbc_other_object_creation

You can see under CREATE TYPE

```
user-defined type definition

<user-defined type definition> ::= CREATE TYPE <user-defined type body>

<user-defined type body> ::= <schema-resolved user-defined type name> [ AS <representation> ]

<representation> ::= <predefined type>

Define a user-defined type. Currently only simple distinct types can be defined without further attributes.
```

mentioning that only predefined type is allowed as the representation of the type.

Reply:
Then lets remove this as this is not what we wanted to test with user defined types

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/291/commits/a24fa36382ca4de8797472c84247e76d65a4734e